src/main/java/org/opensearch/ad/rest/RestSearchAnomalyResultAction.java

```diff
@@ -41,6 +52,27 @@ public RestSearchAnomalyResultAction() {
 
     @Override
     public String getName() {
-        return SEARCH_ANOMALY_DETECTOR_ACTION;
+        return SEARCH_ANOMALY_RESULT_ACTION;
+    }
+
+    @Override
+    protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) throws IOException {
```

Comment:
Is a similar change needed on the delete anomaly result rest action?

Reply:
Discussed with manager, we planned to let user owns the result deletion by themselves. Actually, we rolled over default result index with some strategy like if default result index doc size reach some limit, we will roll over it and create a new index to store new anomaly result. We won't support such roll over for custom result index either. 
So user need to maintain the custom result index by themselves. We will show callout message to warn user and suggest to use ISM plugin to manage custom result index.

Reply:
That's fair, agree the custom index should be owned by the user. May cause some user confusion if they try to run the delete results API on a detector with a custom result index. Might be worth calling that out in the API docs.

Reply:
Yes, will ask tech writer to help add these details to AD doc